File: ZezeJava/ZezexJava/server/src/Game/Fight/Fighter.java
package Game.Fight;

public class Fighter {
	private final BFighterId Id;
	public final BFighterId getId() {
		return Id;
	}
	private final BFighter Bean;
	public final BFighter getBean() {
		return Bean;
	}

	public Fighter(BFighterId id, BFighter bean) {
		this.Id = id;
		this.Bean = bean;
	}

}

File: Sample/server/Game/Fight/Fighter.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Game.Fight
{
    public class Fighter
    {
        public BFighterId Id{ get; }
        public BFighter Bean { get; }

        public Fighter(BFighterId id, BFighter bean)
        {
            this.Id = id;
            this.Bean = bean;
        }

    }
}
